consoles/VNC.pm

```diff
@@ -1139,38 +1023,30 @@ sub _receive_ikvm_encoding {
     }
 }
 
-sub _receive_colour_map {
-    my $self = shift;
-
+sub _receive_colour_map ($self) {
     $self->socket->read(my $map_infos, 5);
     my ($padding, $first_colour, $number_of_colours) = unpack('Cnn', $map_infos);
 
-    for (my $i = 0; $i < $number_of_colours; $i++) {
+    for (0 .. $number_of_colours - 1) {
```

Comment:
`for my $i (0 ...` is cleaner than relying on `$_`.

Reply:
done